change/@azure-msal-browser-927c443a-ae1c-4393-8209-6cc3dde65e47.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

This will require changes in msal-browser-1p so let's make this minor and please make sure to make the required changes in 1p as well
```suggestion
  "type": "minor",
```